### PR TITLE
Feature(common): Interface to view volumes in carousel[SEM-05]

### PR DESCRIPTION
**Pull request by harold-mg**
This is my interface that shows volumes through a carousel, the files (Mocked) are extracted from a JSON file

### CODE REVIEW

**Comment by harold-mg** [comment]
This is my interface that shows volumes through a carousel, the files (Mocked) are extracted from a JSON file

File: src/components/Carousel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <div className={styles.subtitulo}><h2>VOLÚMENES</h2></div>
+      <div className={styles.slider}>
+        
+        <button className={`${styles.arrow} ${currentIndex === 0 ? styles.disabled : ''}`}onClick={handlePrevClick}disabled={currentIndex === 0} >
+          <svg width="39" height="63" viewBox="0 0 39 63" fill="none" xmlns="http://www.w3.org/2000/svg"><g clipPath="url(#clip0_36_446)"><path d="M37.6446 0C25.306 10.3008 12.9653 20.6016 0 31.4258C12.7881 42.1127 25.2883 52.5553 37.7885 63C38.1916 62.6641 38.5969 62.3283 39 61.9924C33.8728 53.3077 28.9294 44.5096 23.5254 35.9972C21.4568 32.7388 21.3881 30.3986 23.4633 27.1162C28.863 18.5667 33.8152 9.74465 38.9468 1.03161C38.5127 0.689192 38.0787 0.344596 37.6446 0Z" fill="#178435"/></g>
+          <defs><clipPath id="clip0_36_446"><rect width="39" height="63" fill="white" transform="matrix(-1 0 0 1 39 0)"/></clipPath></defs>
+          </svg>
+        </button>
+        {mockData.slice(currentIndex, currentIndex + 3).map((item) => (          
+          <div key={item.volume} className={styles.carouselItem}>
```
Comment (Ditmar) [inline src/components/Carousel/index.tsx:28]: this kind of code is considered a bad practice, you need to separate the code into smaller components.
Create a new component called ItemVolumen.tsx an put into the new component this code.

File: src/components/Carousel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    setCurrentIndex((prevIndex) => (prevIndex < mockData.length - 3 ? prevIndex + 1 : 0));
+  };
+
+  return (
+    <div className={styles.carousel}>
+      <div className={styles.subtitulo}><h2>VOLÚMENES</h2></div>
+      <div className={styles.slider}>
+        
+        <button className={`${styles.arrow} ${currentIndex === 0 ? styles.disabled : ''}`}onClick={handlePrevClick}disabled={currentIndex === 0} >
+          <svg width="39" height="63" viewBox="0 0 39 63" fill="none" xmlns="http://www.w3.org/2000/svg"><g clipPath="url(#clip0_36_446)"><path d="M37.6446 0C25.306 10.3008 12.9653 20.6016 0 31.4258C12.7881 42.1127 25.2883 52.5553 37.7885 63C38.1916 62.6641 38.5969 62.3283 39 61.9924C33.8728 53.3077 28.9294 44.5096 23.5254 35.9972C21.4568 32.7388 21.3881 30.3986 23.4633 27.1162C28.863 18.5667 33.8152 9.74465 38.9468 1.03161C38.5127 0.689192 38.0787 0.344596 37.6446 0Z" fill="#178435"/></g>
```
Comment (Ditmar) [inline src/components/Carousel/index.tsx:23]: Another bad practice is not to write the code of the SVG, you need to import directly from the file example.
```
import icon from '../../assets/my-svg.svg'
```
then if you want to use 
```
<icon /> 
```
Only that use this approach.

File: src/components/Carousel/index.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            <a href='post'><img src={item.image} alt={`Volume ${item.volume}`} /></a>
+            <div className={styles.details}>
+              <p className={styles.date}> {item.date}</p>
+              <p className={styles.volume}>Vol: {item.volume}</p>
+            </div>
+          </div>
+        ))}
+      
+        <button className={styles.arrow} onClick={handleNextClick}>
+        <svg width="51" height="63" viewBox="0 0 51 63" fill="none" xmlns="http://www.w3.org/2000/svg"><g clipPath="url(#clip0_36_448)"><path d="M1.7725 0C17.9075 10.3008 34.0454 20.6016 51 31.4258C34.2771 42.1127 17.9307 52.5553 1.58425 63C1.05713 62.6641 0.527117 62.3283 0 61.9924C6.70481 53.3077 13.1692 44.5096 20.2361 35.9972C22.9412 32.7388 23.031 30.3986 20.3172 27.1162C13.2561 18.5667 6.78011 9.74465 0.0695099 1.03161C0.637174 0.689192 1.20484 0.344596 1.7725 0Z" fill="#178435"/></g>
```
Comment (Ditmar) [inline src/components/Carousel/index.tsx:38]: read the comment below

File: src/components/Carousel/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100&display=swap');
```
Comment (Ditmar) [inline src/components/Carousel/styles.module.scss:3]: ok, this is a bad practice, we know the following all projects should use only 3 types of fonts, in this case, Montserrat, roboto, and lato.
That sounds good, you need to import this font into the root of the project before checking if it exits, on the base repo, you can use the optimized font that is a new feature of next13 you can read the documentation here -> https://nextjs.org/docs/app/building-your-application/optimizing/fonts  please update the pr using this approach.

File: src/components/Carousel/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+      }
+    }
+  }
+
+  @include respond-to('large') {
+    .arrow-right {
+      display: none;
+    }
+  }
+  @include respond-to('medium') {
```
Comment (Ditmar) [inline src/components/Carousel/styles.module.scss:99]: Nice

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice pr good job, but fix some comments and you will be approved

**Comment by harold-mg** [comment]
buenos días ingeniero ya realice los cambios, cree el componente ItemVolumen.tsx para maenjar mis archivos MOCK importe los fonts usando la propiedad de NEXTjs y utilice componentes para usar mis íconos.

File: src/components/Carousel/styles.module.scss (hunk 4 above)
Comment (harold-mg) [reply src/components/Carousel/styles.module.scss]: import from the layout page using NEXTjs property

File: src/components/Carousel/index.tsx (hunk 2 above)
Comment (harold-mg) [reply src/components/Carousel/index.tsx]: fixed, icons used via a component tag

**Comment by olmedo90** [comment]
image

It is not according to the figma, check the size and color of the date and year of the items.

**Review by Ditmar — APPROVED** [review]

**Pull request merged**